akka-docs/src/main/paradox/stream/operators/ActorFlow/ask.md

```diff
@@ -18,8 +18,12 @@ This operator is included in:
 
 ## Signature
 
-@@signature [ActorFlow.scala](/akka-stream-typed/src/main/scala/akka/stream/typed/scaladsl/ActorFlow.scala) { #ask }
-
+@apidoc[ask](FlowOpsMat) { scala="#ask[S](parallelism:Int)(ref:akka.actor.ActorRef)(implicittimeout:akka.util.Timeout,implicittag:scala.reflect.ClassTag[S]):FlowOps.this.Repr[S]" scala="#ask[S](ref:akka.actor.ActorRef)(implicittimeout:akka.util.Timeout,implicittag:scala.reflect.ClassTag[S]):FlowOps.this.Repr[S]" }
+@apidoc[ask](GraphDSL$$Implicits$$PortOps) { scala="#ask[S](parallelism:Int)(ref:akka.actor.ActorRef)(implicittimeout:akka.util.Timeout,implicittag:scala.reflect.ClassTag[S]):FlowOps.this.Repr[S]" scala="#ask[S](ref:akka.actor.ActorRef)(implicittimeout:akka.util.Timeout,implicittag:scala.reflect.ClassTag[S]):FlowOps.this.Repr[S]" }
+@apidoc[ask](FlowOps) { scala="#ask[S](parallelism:Int)(ref:akka.actor.ActorRef)(implicittimeout:akka.util.Timeout,implicittag:scala.reflect.ClassTag[S]):FlowOps.this.Repr[S]" scala="#ask[S](ref:akka.actor.ActorRef)(implicittimeout:akka.util.Timeout,implicittag:scala.reflect.ClassTag[S]):FlowOps.this.Repr[S]" }
+@apidoc[ask](Source) { scala="#ask[S](parallelism:Int)(ref:akka.actor.ActorRef)(implicittimeout:akka.util.Timeout,implicittag:scala.reflect.ClassTag[S]):FlowOps.this.Repr[S]" scala="#ask[S](ref:akka.actor.ActorRef)(implicittimeout:akka.util.Timeout,implicittag:scala.reflect.ClassTag[S]):FlowOps.this.Repr[S]" java="#ask(akka.actor.ActorRef,java.lang.Class,akka.util.Timeout)" java="#ask(int,akka.actor.ActorRef,java.lang.Class,akka.util.Timeout)" }
+@apidoc[ask](SubFlow) { scala="#ask[S](parallelism:Int)(ref:akka.actor.ActorRef)(implicittimeout:akka.util.Timeout,implicittag:scala.reflect.ClassTag[S]):FlowOps.this.Repr[S]" scala="#ask[S](ref:akka.actor.ActorRef)(implicittimeout:akka.util.Timeout,implicittag:scala.reflect.ClassTag[S]):FlowOps.this.Repr[S]" }
+@apidoc[ask](Flow) { scala="#ask[S](parallelism:Int)(ref:akka.actor.ActorRef)(implicittimeout:akka.util.Timeout,implicittag:scala.reflect.ClassTag[S]):FlowOps.this.Repr[S]" scala="#ask[S](ref:akka.actor.ActorRef)(implicittimeout:akka.util.Timeout,implicittag:scala.reflect.ClassTag[S]):FlowOps.this.Repr[S]" java="#ask(akka.actor.ActorRef,java.lang.Class,akka.util.Timeout)" java="#ask(int,akka.actor.ActorRef,java.lang.Class,akka.util.Timeout)" }
 @@@
```

Comment:
I am considering a manual trim of the generated output so the more complex cases (such as the `ask` operator above) need to be migrated manually.